### DISCUSSION TITLE
AnalyzerEngine failing to parse and analyze

### DISCUSSION THREAD

**Discussion by amitt117** [microsoft/presidio#1003] [Q&A]
My python code below is failing to analyse. 
If I am running this code 10-15 times , it is running successfully some time , and  failing occasionaly around 4-5 times.
When it fails it does not execute the code inside except block or  even finally block. The code just exits abruptly without any error.
If I change the input text then it is not failing. 
Can some one help me to understand what is it that I am doing wrong.
I am using presidio 2.2.29.
Thanks for help. 
```
from presidio_analyzer import AnalyzerEngine

analyzer = AnalyzerEngine()
if __name__ == "__main__":
    try:     
        text = 'the source of the hue river is the clear spring'
        results = analyzer.analyze(text=text, language="en")
        print (results)
    except Exception as e:
        print("Exception Occured")
    finally:
       print("Job completed") 
    print('Finally Job Completed')
```

**Comment by omri374** [comment]
Hi @amitt117, I cannot reproduce the error you're getting. If you can provide any stacktrace or any information about the error, that would be helpful.
